ignite/handlers/lr_finder.py

```diff
@@ -87,7 +87,7 @@ def _run(
         optimizer: Optimizer,
         output_transform: Callable,
         num_iter: int,
-        start_lr: float,
+        start_lr: Union[None, float, List[float]],
         end_lr: float,
```

Comment:
Please change end_lr type hint as well:
```suggestion
        end_lr: Union[float, List[float]],
```